lighthouse-core/test/audits/load-fast-enough-for-pwa-test.js

```diff
@@ -78,4 +78,27 @@ describe('PWA: load-fast-enough-for-pwa audit', () => {
     expect(result.rawValue).toBeGreaterThan(2000);
     expect(Math.round(result.rawValue)).toMatchSnapshot();
   });
+
+
+  it('override with simulated result fails a bad simulated TTI value', async () => {
+    const topLevelTasks = [
+      {ts: 1000, duration: 1000},
+      {ts: 3000, duration: 1000},
+      {ts: 5000, duration: 1000},
+      {ts: 9000, duration: 1000},
+      {ts: 12000, duration: 1000},
+      {ts: 14900, duration: 1000},
+    ];
+    const longTrace = createTestTrace({navigationStart: 0, traceEnd: 20000, topLevelTasks});
+
+    const artifacts = {
+      traces: {defaultPass: longTrace},
+      devtoolsLogs: {defaultPass: devtoolsLog},
+    };
+
+    const settings = {throttlingMethod: 'provided', throttling: {rttMs: 40, throughput: 100000}};
+    const result = await FastPWAAudit.audit(artifacts, {settings, computedCache: new Map()});
+    expect(result.displayValue).toContain('Interactive on simulated mobile network at %d\xa0s');
+    expect(result.rawValue).toBeGreaterThan(10000);
```

Comment:
@patrickhulce I expected this test to work with the same config as `fails a bad TTI value` but I had to bump up the durations to get it to be over 10s.  Thoughts on why?

Reply:
without more events/network requests there's no way for the simulator to know that the events couldn't just happen all one after the other, i.e. under simulation it's going to mostly ignore the timestamps in the original trace and just have 6 100ms tasks one after the other.

If you want to simplify this and make it a little more clear, I'd probably move this down to just a single 5s task in the trace. With the multiplier it should be longer than 10s guaranteeing late TTI, without the multiplier it would be well under 10s.